### DISCUSSION TITLE
Livewire 3 Dark mode Issue

### DISCUSSION THREAD

**Discussion by mai1237** [livewire/livewire#6732] [1. Help]
Laravel with laravel/breeze and Livewire 3 installed
the code below works if the page has no livewire components
with livewire components the button does not work
when i go back to page with no livewire components still not working until i refresh the page
any idea ?

resources\views\layouts\app.blade.php

    <body class="font-sans antialiased"
    x-cloak
    x-data="{darkMode: localStorage.getItem('dark') === 'true'}"
    x-init="$watch('darkMode', val => localStorage.setItem('dark', val))"
    x-bind:class="{'dark': darkMode}"
    >
        <div class="min-h-screen bg-gray-100 dark:bg-gray-900">
            @include('layouts.navigation')
......

resources\views\layouts\navigation.blade.php

                
                <div class="shrink-0 flex items-center">
                    <a href="{{ route('dashboard') }}">
                        <x-application-logo class="block h-9 w-auto fill-current text-gray-800 dark:text-gray-200" />
                    </a>

<div class="p-2">
<button x-cloak x-on:click="darkMode = !darkMode;" >
    <x-pk.icons.heroicon-s-moon x-show="!darkMode" class="p-2 ml-3 w-8 h-8 text-gray-700 bg-gray-100 rounded-md transition cursor-pointer hover:bg-gray-200" />
    <x-pk.icons.heroicon-s-sun x-show="darkMode" class="p-2 ml-3 w-8 h-8 text-gray-100 bg-gray-700 rounded-md transition cursor-pointer dark:hover:bg-gray-600" />
</button>
</div>
</div>

**Comment by joshhanley** [comment]
@mai1237 Breeze at the moment isn't compatible with V3 out of the box. As Alpine is now built into Livewire, you need to edit your `app.js` to remove Alpine as per the instructions in the V3 upgrade guide.

Hope this helps!

**Reply by mai1237** [reply]
Some more help please
